### DISCUSSION TITLE
Add support for input broadcasting

### DISCUSSION THREAD

**Discussion by Radiergummi** [ghostty-org/ghostty#3959] [Feature Requests, Ideas] [answered]
iTerm 2 has support for *input broadcasting*, that is, sending keystrokes to all split panes or tabs simultaneously. This comes in *very* handy when you need to update a bunch of servers at once, for example, and has grown very dear to me. Would that be something that I may kindly request to place on the wishlist for Ghostty, too?

I would expect this feature to provide keyboard bindings to:
 - Send input to all panes in the current tab
 - Send input to all panes in all open tabs
 - Only send input to the currently selected pane

**Comment by mitchellh** [comment, marked as answer]
Dupe. Please search but I saw this yesterday!

**Reply by Radiergummi** [reply]
Sorry. I actually searched a bit, but must have overlooked it—my bad.
